src/css/popup.css

```diff
@@ -271,21 +271,63 @@ main-panel {
   font-weight: 700;
 }
 
+.panel-status {
+  display: flex;
+  justify-content: space-between;
+  padding: 8px 15px;
+  line-height: 150%;
+  font-size: 1rem;
+  border-bottom: 1px solid var(--relayGrey20);
+}
+
 .aliases-remaining {
-  font-family: var(--relayMetropolis);
-  font-weight: 300;
-  font-size: 14px;
   margin: 0;
-  padding: 8px;
+  font-family: 'InterUI';
   background: white;
-  border-bottom: 1px solid var(--relayGrey20);
+}
+
+.premium-cta {
+  margin: 0;
+  font-family: 'InterUI';
+  color: var(--relayBlue3);
+  font-weight: 700;
+  text-decoration: none;
+}
+
+.premium-cta a {
+  margin: 0 !important;
 }
 
 .num-aliases-remaining,
 .max-num-aliases {
   font-weight: 700;
 }
 
+/*upgrade banner*/
+
+.upgrade-banner-wrapper {
+  background: var(--relayGrey20);
+  border-radius: 8px;
+  padding: 10px;
+  margin-top: 10px;
+  display: flex;
```

Comment:
Protocol token value callout. Could we use `8px` or `16px` here?

Reply:
Changed to 8px!